lib/bike_brigade_web/live/campaign_live/messaging_form_component.html.heex

```diff
@@ -22,14 +22,17 @@
     autocomplete="off" phx-submit="save">
   <div class="flex flex-row mt-2 space-x-3">
     <div class="w-1/2">
-      <%= label f, :body, class: "block text-sm font-medium leading-5 text-gray-700" do %>
-        Text Message
+      <%= label f, :text, class: "block text-sm font-medium leading-5 text-gray-700" do %>
+        Text Message 
+        <C.button id="emoji-button" phx-hook="EmojiButtonHook" data-input-id={input_id(f, :textbody)} type="button" color={:white} size={:xxsmall}>
```

Comment:
IDs for html elements need to be unique. You have two different elements with the same ID `emoji-button`. This is causing some of your problems